internal/services/containerapps/helpers/container_apps.go

```diff
@@ -1007,6 +1014,213 @@ func ContainerAppContainerSchemaComputed() *pluginsdk.Schema {
 	}
 }
 
+type BaseContainer struct {
+	Name             string                 `tfschema:"name"`
+	Image            string                 `tfschema:"image"`
+	CPU              float64                `tfschema:"cpu"`
+	Memory           string                 `tfschema:"memory"`
+	EphemeralStorage string                 `tfschema:"ephemeral_storage"`
+	Env              []ContainerEnvVar      `tfschema:"env"`
+	Args             []string               `tfschema:"args"`
+	Command          []string               `tfschema:"command"`
+	VolumeMounts     []ContainerVolumeMount `tfschema:"volume_mounts"`
+}
+
+func InitContainerAppContainerSchema() *pluginsdk.Schema {
+	return &pluginsdk.Schema{
+		Type:     pluginsdk.TypeList,
+		Optional: true,
+		MinItems: 1,
+		Elem: &pluginsdk.Resource{
```

Comment:
The docs list this as a `ForceNew` property, but this is not the case here, should this be?
```suggestion
	return &pluginsdk.Schema{
		Type:     pluginsdk.TypeList,
		Optional: true,
		ForceNew: true,
		MinItems: 1,
		Elem: &pluginsdk.Resource{
```

Reply:
Though I commited this suggested change, I want to revert it.
I should update the docs to remove the sentence `Changing this forces a new resource to be created.`.
Though I reffered the code and document of `azure_container_group` to create this PR,  I find the difference of coding way between them.